### ISSUE TITLE
FOSFacebookBundle dont save user and login with fbid on username

### ISSUE THREAD

**Issue by cirovargas** [FriendsOfSymfony/FOSFacebookBundle#255]
Can help me

http://stackoverflow.com/questions/16341399/fosfacebookbundle-dont-save-user-and-login-with-fbid-on-username

**Comment by ftassi** [comment]
Question on stackoverflow has been removed, is this issue solved ?

**Comment by cirovargas** [comment]
No, I made a controller to register the user, I'm using only fosuserbundle now

**Comment by ftassi** [comment]
Ho the user is saved depends on the FacebookProvider that you have imlpemented in yout application. There is an example of a FacebookProvider in the documentation. If you are using the code from the doc you're storing facebook id as the username of your user.

You can change your FacebookProvider to store whatever you want in your user.
